src/factories/Axis.ts

```diff
@@ -7,9 +7,12 @@ module n3Charts.Factory {
     public scale: D3.Scale.Scale;
     public axis: D3.Svg.Axis;
 
-    constructor(public name: string) {
+    constructor(public side: string) {
       super();
 
+      if (['y', 'x'].indexOf(side) < 0) {
```

Comment:
Yeah, me too, and that's one of the few things that I really liked in CoffeeScript, you could write `throw new TypeError('') if side not in ['x', 'y']` !